src/EncryptedRow.php

```diff
@@ -83,9 +84,10 @@ class EncryptedRow
     public function __construct(
         CipherSweet $engine,
         #[\SensitiveParameter]
-        string $tableName,
-        bool $useTypedIndexes = false
-    ) {
+        string      $tableName,
+        bool        $useTypedIndexes = false
+    )
+    {
```

Comment:
Your IDE is changing the code style. Can you please revert these?

Reply:
I pushed a commit and reverted all the IDE code style back to how it was